### PR TITLE
Speed up test for MD5 implementation equivalence

### PR DESCRIPTION
**Pull request by BiggerNoise**
Test was performing 1M iterations over a 10K byte field. This was taking
60-90s depending on the machine. I dropped this to 2K iterations.

### CODE REVIEW

**Comment by BiggerNoise** [comment]
Fixes #289

File: product/roundhouse.tests/cryptography/CryptographicServiceSpecs.cs [hunk 1]
```diff
@@ -55,7 +55,7 @@ public void should_be_able_to_pass_text_and_get_back_a_base_64_hash_of_the_text(
         [TestFixture]
         public class when_using_an_unofficial_md5_implementation
         {
-            const int passes = 1000000;
+            const int passes = 2000;
```
Comment (erikbra) [inline product/roundhouse.tests/cryptography/CryptographicServiceSpecs.cs]: Nice perf improvement! ;)

Comment (ferventcoder) [reply product/roundhouse.tests/cryptography/CryptographicServiceSpecs.cs]: 😂

Comment (BiggerNoise) [reply product/roundhouse.tests/cryptography/CryptographicServiceSpecs.cs]: I had to sweat for days micro-optimizing that code. Anything for my peeps.

**Pull request merged**